hydra/app/jobs/generate_pdf_thumbs_job.rb

```diff
@@ -3,44 +3,20 @@ class GeneratePdfThumbsJob < ApplicationJob
 
   queue_as :import
 
-  def perform(identifier)
-    # find record
-    record = Acda.where(identifier: identifier).first
+  def perform(id, download_path)
```

Comment:
See the above conversation regarding renaming the parameter.

Reply:
see above